contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -130,9 +130,10 @@ contract OETHVaultCore is VaultCore {
         }
 
         // Amount excluding fees
-        uint256 amountMinusFee = _calculateRedeemOutputs(_amount)[
-            wethAssetIndex
-        ];
+        // No fee for the strategist, makes it easier to do operations
```

Comment:
nit: comment should also include the governor

Reply:
Fixed. thanks